EpicLoot/MagicItemEffects/MultiShot.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using HarmonyLib;
+using JetBrains.Annotations;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace EpicLoot.MagicItemEffects
+{
+    [HarmonyPatch]
+    public static class MultiShot
+    {
+        [HarmonyPatch(typeof(Attack), "FireProjectileBurst")]
+        class FireProjectileBurstPatch
+        {
+            public static void Prefix(Attack __instance)
+            {
+                var player = (Player)__instance.m_character;
```

Comment:
I would do a null check here on __instance and early out if null. Remember to do proper null comparisons (object == null) for Unity objects.